src/Symfony/Contracts/Deprecation/deprecated.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Triggers a deprecation.
+ *
+ * As recommended for prod, turn the "zend.assertions" ini setting to 0 or -1 to disable deprecation notices.
```

Comment:
I think we could explain here that turning this setting to 0 actually disable all assertions and since the deprecation is "wrapped" in an assertion, it will also disable deprecations.

Reply:
I think there is zero reasons to do so.
The side that must take care of this is the author, not the users.
If authors use `assert()` they MUST be prepared to it being disabled in prod, that's the ONLY reason why this language construct exists.